pkg/server/api/api.go

```diff
@@ -24,3 +25,16 @@ func ProtoFromID(id spiffeid.ID) *types.SPIFFEID {
 		Path:        id.Path(),
 	}
 }
+
+// StringValueFromProto converts a SPIFFE ID from the given spiffeid.ID to
+// *wrappers.StringValue
+func StringValueFromProto(spiffeID *types.SPIFFEID) (*wrappers.StringValue, error) {
```

Comment:
How would you feel about naming this `SPIFFEIDToStringValue` or `StringValueFromSPIFFEID`?

Other "ToProto/FromProto" methods are converting to/from the old and new versions of something (i.e. entry, etc) but that isn't the case here.

Reply:
True! I will make the change.